.yarnrc.yml

```diff
@@ -0,0 +1,15 @@
+enableScripts: false
```

Comment:
Could you update this to disable telemetry? That was added recently in the module template